index.js

```diff
@@ -1,32 +1,8 @@
-import Library from './modules/library.js';
-import showTime from './modules/dateTime.js';
-// eslint-disable-next-line no-unused-vars
-const library = new Library();
+import BookCollection from './modules/bookCollection.js';
+import showTime from './modules/date.js';
 
-const headerLinks = document.querySelector('#header');
-const popList = document.querySelector('#pop-list');
-const listOfBooks = document.querySelector('#book-lists');
-const popAdded = document.querySelector('#pop-added');
-const addBook = document.querySelector('.add-book-section');
-const popContact = document.querySelector('#pop-contact');
-const showContact = document.querySelector('.contact');
-
-headerLinks.addEventListener('click', (e) => {
-  e.preventDefault();
-  const currentLink = e.target;
-  if (currentLink === popList) {
-    listOfBooks.classList.add('show-list');
-    addBook.classList.remove('show-add-menu');
-    showContact.classList.remove('show-contact');
-  } else if (currentLink === popAdded) {
-    addBook.classList.add('show-add-menu');
-    listOfBooks.classList.remove('show-list');
-    showContact.classList.remove('show-contact');
-  } else if (currentLink === popContact) {
-    showContact.classList.add('show-contact');
-    listOfBooks.classList.remove('show-list');
-    addBook.classList.remove('show-add-menu');
-  }
+document.addEventListener('DOMContentLoaded', () => {
+  // eslint-disable-next-line no-unused-vars
+  const bookCollection = new BookCollection();
```

Comment:
- [ ] You have done perfect work, however, we have a few points of improvement, talking about linter, it helps us clean and standardizes our code regarding best practices and DRY method, to say so, I believe it is not a good practice to disable it in our code, I would strongly recommend you fix it up and remove the disabling lines as this will contribute to your professionalism and gain some good ways of writing a clean code in the near future, what do you think about it? 😃

Reply:
Hello @joel-kalema
I really found it hard to resolve this issue and spent more hours on it and you know we have more tasks in this block could you suggest to me how can be fixed?